client/layout/masterbar/drafts.jsx

```diff
@@ -148,6 +148,13 @@ export default connect( state => {
 		draftsQuery: draftsQuery,
 		draftCount: myPostCounts && myPostCounts.draft,
 		selectedSite: site,
-		recordTracksEvent,
 	};
-} )( localize( MasterbarDrafts ) );
+};
+
+const mapDispatchToProps = dispatch => ( {
+	recordDraftSelected: () => {
+		dispatch( recordTracksEvent( 'calypso_masterbar_publish_button_draft_selected' ) );
+	},
```

Comment:
fun fact, these curly brackets aren't needed
if that doesn't make sense just ping me in Slack :wink:

Reply:
@dmsnell I put them in because I had two lines here at one point. When I removed the second line I decided to keep them braces because I think it's a) easier to read and b) will cause fewer changed lines in the future if we decide to add another line again — similar to how even for the last item / line in an array we would add a comma (I would, and I've seen many others who would — not sure whether it's official convention though). 

Can we leave them in please?

Reply:
> Can we leave them in please?

you are free to ignore any review comments about anything :wink:

> will cause fewer changed lines in the future if we decide to add another line again

just use your good judgment. we just want to make sure that we don't make choices we otherwise wouldn't make on account of the fact that there could be unforeseen changes in the future. in this case, it's sole purpose is recording metrics.